src/main/java/com/ft/methodearticleinternalcomponentsmapper/transformation/InternalComponentsMapper.java

```diff
@@ -190,7 +186,7 @@ private String validateBody(final TransformationMode mode,
             return EMPTY_VALIDATED_BODY;
         }
 
-        if (Type.CONTENT_PACKAGE.equals(type)) {
```

Comment:
type can be null, which would result in a NPE, keeping constants on the left side of equals avoids that